css/main.css

```diff
@@ -234,22 +198,14 @@ input {
 	border-radius: 6px;
 }
 
-.hero {
-	background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url("https://bit.ly/2N5kvOq");
-	height: 400px;
-	background-position: center;
-	background-repeat: no-repeat;
-	background-size: cover;
-	position: relative;
-}
-
 .hero-text {
 	text-align: center;
 	position: relative;
 	top: 50%;
 	left: 50%;
 	transform: translate(-50%, -50%);
 	color: white;
+	text-shadow: 2px 2px black;
```

Comment:
OK but we still want better shadowing. use `3px 3px 6px black` for this. That's it and you're good I think.